crates/zed/resources/flatpak/template.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+            <li>AI Integration: Integrates GitHub Copilot and GPT-4 for natural language code generation.</li>
+        </ul>
+    </description>
+
+    <launchable type="desktop-id">app_id.desktop</launchable>
+
+    <branding>
+        <color type="primary" scheme_preference="light">branding_light</color>
+        <color type="primary" scheme_preference="dark">branding_dark</color>
+    </branding>
+
+    <content_rating type="oars-1.1">
+        <content_attribute id="social-chat">intense</content_attribute>
+        <content_attribute id="social-audio">intense</content_attribute>
+    </content_rating>
+
+    <url type="homepage">https://zed.dev</url>
+    <url type="bugtracker">https://github.com/zed-industries/zed/issues</url>
+    <url type="faq">https://zed.dev/faq</url>
+    <url type="help">https://zed.dev/docs/getting-started</url>
+    <url type="contact">https://zed.dev/docs/feedback-and-support</url>
+    <url type="vcs-browser">https://github.com/zed-industries/zed</url>
+    <url type="contribute">https://github.com/zed-industries/zed/blob/main/CONTRIBUTING.md</url>
+
+    <supports>
+        <internet>offline-only</internet>
+    </supports>
+    <recommends>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <display_length compare="ge">768</display_length>
+    </recommends>
+
+    <screenshots>
+        <screenshot type="default">
+            <caption>Zed with a large project open, showing language server and gitblame support</caption>
+            <image type="source" width="1122" height="859" xml:lang="en">https://raw.githubusercontent.com/someone13574/zed/flatpak-2/crates/zed/resources/flatpak/screenshots/screenshot-0.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Zed with a file open and a channel message thread in the right sidebar</caption>
+            <image type="source" width="1122" height="859" xml:lang="en">https://raw.githubusercontent.com/someone13574/zed/flatpak-2/crates/zed/resources/flatpak/screenshots/screenshot-1.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Example of a channel's shared document</caption>
+            <image type="source" width="1122" height="859" xml:lang="en">https://raw.githubusercontent.com/someone13574/zed/flatpak-2/crates/zed/resources/flatpak/screenshots/screenshot-2.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Zed's extension list</caption>
+            <image type="source" width="1122" height="859" xml:lang="en">https://raw.githubusercontent.com/someone13574/zed/flatpak-2/crates/zed/resources/flatpak/screenshots/screenshot-3.png</image>
+        </screenshot>
+        <screenshot>
+            <caption>Theme switcher UI and example theme</caption>
+            <image type="source" width="1122" height="859" xml:lang="en">https://raw.githubusercontent.com/someone13574/zed/flatpak-2/crates/zed/resources/flatpak/screenshots/screenshot-4.png</image>
```

Comment:
I guess the links might vanish when the fork is deleted?

Reply:
Yep. Plus I want it to update if somebody changes it.